### PR TITLE
Minor documentation updates

### PR DESCRIPTION
**Pull request by vparpoil**
- Generation of settings in the UI for new apps
- MAIL_URL env variable
- Sorting of the apps by name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +25,9 @@ docker run -d --name meteor-apm-server \
 
 This can be useful for running an instance quickly in your own environment with orchestration.
 
-### Setup steps
+### Setup steps using MUP
+
+Keep in mind when using MUP: [your user account on the server must be sudoer without password](http://meteor-up.com/docs.html#ssh-based-authentication-with-sudo)
```
Comment (lmachens) [inline README.md:30]: Why is this necessary? It works for me without.

File: mup-placeholder.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ module.exports = {
     env: {
       ROOT_URL: 'https://apm.YOUR_DOMAIN.com',
       MONGO_URL: 'mongodb://mongodb/meteor',
-      MONGO_OPLOG_URL: 'mongodb://mongodb/local'
+      MONGO_OPLOG_URL: 'mongodb://mongodb/local',
+      MAIL_URL: ''
```
Comment (lmachens) [inline mup-placeholder.js:24]: Do you know if mailing works?

File: client/components/app/partials/noData.html [hunk 3]
```diff
@@ -4,40 +4,15 @@
...
     <p>1. {{i18n 'app.config_install_pkg'}}</p>
-    <pre><code>meteor add meteorhacks:kadira
-
-  // for Meteor 0.8.3 and older versions
-  mrt add kadira</code></pre>
+    <pre><code>meteor add lmachens:kadira</code></pre>
 
     <p>2. {{{i18n 'app.config_connect_code'}}}</p>
     <div id="no-data-code-wrap">
-      <ul class="nav nav-tabs" role="tablist">
```
Comment (lmachens) [inline client/components/app/partials/noData.html]: Actually I like to have the option to set the apm settings via code or env. So I would not remove these tabs here and better add the apm engine url to the other tabs too:
Code:
`Kadira.connect('{{state$appId}}', '{{state$secret}}', { endpoint:  apmEngineUrl });`
For env:
KADIRA_OPTIONS_ENDPOINT (https://github.com/lmachens/kadira/blob/master/lib/environment_variables.js#L67)

**Review by lmachens — CHANGES_REQUESTED** [review]
Thank you for your pr! 
I added some comments

File: mup-placeholder.js (hunk 2 above)
Comment (vparpoil) [reply mup-placeholder.js:24]: Yes, mailing works (I tested invinting users to an application)

File: README.md (hunk 1 above)
Comment (vparpoil) [reply README.md:30]: the links give the details about this. maybe your username is root ?

Comment (lmachens) [reply README.md:30]: no, but doesn't matter. this comment won't hurt :)

**Comment by vparpoil** [comment]
I just updated with the two other options to connect an app, thanks for your feedback, and sorry for the bad indentation on commit 023d8c7, I fixed it

**Review by lmachens — APPROVED** [review]
thx! I will add prettier soon

**Comment by lmachens** [comment]
I added `apmEngineUrl` to the settings dialog too. 
see https://github.com/lmachens/meteor-apm-server/commit/e02f924bd27381b773798ff18824d93cd1a2532f
